tests/BulbTests.py

```diff
@@ -0,0 +1,133 @@
+import mockito
+import unittest
+import Bulb
+
+class BulbTests(unittest.TestCase):
+
+    def setUp(self):
+        self.home_sensor = mockito.mock()
+        self.desk_sensor = mockito.mock()
+        self.bed_sensor = mockito.mock()
+        self.light_sensor = mockito.mock()
+
+    def setReturnValues(self, bools):
+        mockito.when(self.home_sensor).isHome().thenReturn(bools[0])
+        mockito.when(self.desk_sensor).isAround().thenReturn(bools[1])
+        mockito.when(self.bed_sensor).isAround().thenReturn(bools[2])
+        mockito.when(self.light_sensor).isOn().thenReturn(bools[3])
+
+    def test_kitchenLampWastedNotAround(self):
+        self.setReturnValues((True, True, False, True))
+
+        bulb = Bulb.Bulb('Kitchen',self.light_sensor,[self.desk_sensor,self.bed_sensor],self.home_sensor,Bulb.kitchenLampWasted)
```

Comment:
Why do we need to say `Bulb.Bulb(..)` when constructing one? Why not just `Bulb(..)` ?